apps/i18n/pythonlab/en_us.json

```diff
@@ -0,0 +1,3 @@
+{
+  "inputFailed": "We're sorry, input is unavailable at this time. Input may not work in a private or incognito window. If you are in a standard browser window, please try refreshing the page. If the issue persists, contact support@code.org."
```

Comment:
Maybe specify that it's the input function?
`We're sorry, the input() function is ...'

Reply:
This could also happen when reading from sysin (I know we aren't using it in the curriculum, but we are patching it and a user could discover it).